care.js

```diff
@@ -310,19 +312,33 @@ function catSay(text) {
 }
 
 function getAnsiArt(textToSay) {
-  var artFileRegex = /.ansi$/;
-
-  // If config.say is custom art file path, then return custom art
-  if (artFileRegex.test(config.say)) {
-    return ansiArt.get({ filePath: config.say, speechText: textToSay });
-  }
-
-  switch (config.say) {
-    case 'bunny' : return bunnySay(textToSay);
-    case 'llama' : return llamaSay(textToSay);
-    case 'cat'   : return catSay(textToSay);
-    case 'yeoman': return yosay(textToSay);
-    default : return ansiArt.get({ artName: config.say, speechText: textToSay });
+  if (/.ansi$/.test(config.say)) {
+    // if SAY ends with ".ansi" use the 'ansi-art' ligrary
+    if (config.say === path.basename(config.say)) {
```

Comment:
can we move this into a function called `ansiSay`, to clean it up and make this entire function more readable/

Reply:
I'll see what I can do...

Reply:
I moved the function `getAnsiArt` and all its dependencies to its own file.
This simplifies `care.js` and hopefully makes it easier for newbies to add new artworks.

@notwaldorf I hope you like it.